src/builtins/BuiltinTemporal.cpp

```diff
@@ -237,6 +237,24 @@ static Value builtinTemporalPlainDateInLeapYear(ExecutionState& state, Value thi
     return TemporalCalendarObject::calendarInLeapYear(state, thisValue.asObject()->asTemporalPlainDateObject()->calendar(), thisValue);
 }
 
+static Value builtinTemporalPlainDatePrototypeAdd(ExecutionState& state, Value thisValue, size_t argc, Value* argv, Optional<Object*> newTarget)
```

Comment:
Just a minor request,
The length of function name is quite a long and we usually don't add `Prototype` word to function of prototype object.
Please shorten this kind of function name including other prototype functions.

Reply:
In a future patch i will rename the functions.